lib/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -449,39 +468,22 @@ struct ElementwiseInlineAsmOpConversion
           unpackI32(subOperands, argTy, rewriter, loc, getTypeConverter()));
     }
 
-    // Although we ensure that all operands and results to this op have the same
-    // encoding, MMA layouts have a different physical ordering depending on the
-    // bit width of the underlying element.
-    //
-    // Thus if the inputs to the inline asm op are MMA with different widths, we
-    // need to reorder them so we iterate over the operands' elements in the
-    // same logical order.
-    for (unsigned i = 0; i < unpackedOperands.size(); ++i) {
-      unpackedOperands[i] = reorderValues(
```

Comment:
Hm, where did this call to reorderValues go?

Reply:
it was not correct, removed it